src/site/xdoc/downloads.xml

```diff
@@ -89,6 +89,28 @@ under the License.
       </td>
       <td />
     </tr>
+    <tr>
+      <td style="test-align: left">
+        1.5.0
+      </td>
+      <td style="test-align: left">
+        2019/10/14
+      </td>
+      <td style="test-align: left">
+        <a href="https://apache.org/dist/hbase/1.5.0/compat-check-report.html">1.4.10 vs 1.5.0</a>
+      </td>
+      <td style="test-align: left">
+        <a href="https://github.com/apache/hbase/blob/rel/1.5.0/CHANGES.txt">Changes</a>
+      </td>
+      <td style="test-align: left">
+        <a href="https://s.apache.org/hbase-1.5.0-jira-release-notes">Release Notes</a>
+      </td>
+      <td style="test-align: left">
+        <a href="https://www.apache.org/dyn/closer.lua/hbase/1.5.0/hbase-1.5.0-src.tar.gz">src</a> (<a href="https://apache.org/dist/hbase/1.5.0/hbase-1.5.0-src.tar.gz.sha512">sha512</a> <a href="https://apache.org/dist/hbase/1.5.0/hbase-1.5.0-src.tar.gz.asc">asc</a>) <br />
+        <a href="https://www.apache.org/dyn/closer.lua/hbase/1.5.0/hbase-1.5.0-bin.tar.gz">bin</a> (<a href="https://apache.org/dist/hbase/1.5.0/hbase-1.5.0-bin.tar.gz.sha512">sha512</a> <a href="https://apache.org/dist/hbase/1.5.0/hbase-1.5.0-bin.tar.gz.asc">asc</a>)
+      </td>
+      <td />
+    </tr>
     <tr>
       <td style="test-align: left">
         1.4.12
```

Comment:
Shouldn't this one be also under `stable-1 releases` (see line 134)?

Reply:
As far as I know stable-1 release is 1.4.x and 1.5 does not have the stable pointer.
This points to 1.4.12: https://apache.org/dist/hbase/stable-1/